sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1276,6 +1276,16 @@ val GPU_COREDUMP_PIPE_PATTERN = conf("spark.rapids.gpu.coreDump.pipePattern")
     .booleanConf
     .createWithDefault(false)
 
+  val ORC_STRIPE_SIZE_ROWS = conf("spark.rapids.sql.format.orc.write.stripeSizeRows")
+    .doc("When set to a valid value(no less than 512, as specified in cudf), " +
```

Comment:
maybe we can add a check like `.checkValue(v >= 512)` since there is a known valid value range.

Reply:
Nit:Add `TEST_` prefix for this variable like `TEST_RETRY_OOM_INJECTION_MODE`, and add comments like: "This is really just for testing."